lib/surface_site_web/live/components_basics.ex

```diff
@@ -214,9 +212,7 @@ defmodule SurfaceSiteWeb.ComponentsBasics do
                   </#Raw>
                 </footer>
               </div>
-
               Example 2
-
               <div class="card dark">
                 <footer class="card-footer">
```

Comment:
It's surprising to me that the formatter is removing these newlines.

Looks like a bug in the collapsing of whitespace surrounding text nodes. I'll investigate further.